src/index.html

```diff
@@ -11,7 +11,213 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label 
+          for="surename" 
+          class="fieldsetItem"
+        >
+          Surname:
+          <input 
+            type="text" 
+            name="surename" 
+            autocomplete="off"
+            id="surename"
+          >
+        </label>
```

Comment:
no needs to use for/id whenever you wrap an input with a label. Apply all over the page
```suggestion
        <label 
          class="fieldsetItem"
        >
          Surname:
          <input 
            type="text" 
            name="surename" 
            autocomplete="off"
          >
        </label>
```